deployment/teams/team-j/dev/java-application.yaml

```diff
@@ -14,6 +14,6 @@ spec:
         - type: path-based-ingress
           properties:
             domain: "*.amazonaws.com"
-            rewritePath: "/app" 
```

Comment:
The `app` context path worked, why are you changing this? 
With java-app context path you don't need `rewritePath` at all.

Reply:
its making it consistent and simpler